README.md

```diff
@@ -1,2 +1,139 @@
-# CODE9000
-Urban Data Birds: Providing a new haven for urban biodiversity.
+\#Code9000
+===================
+----------
+
+\#Code9000 is a project in which Digipolis Ghent and the City of Ghent experiment with open data to create new open source proof-of-concepts. We’ve been doing this since 2011 and love to work with technical, business and communication students to co-create new tools. Students get real cases to work on and move beyond the theoretical and we get receive new learnings and code to test and iterate upon. This time we want to build an IOT use case, combining hardware and software to feed our data portal rather than using the data we already have.  
+
+
+Getting Started
+-------------
+----------
+####  Requirements
+
+ - php  (^7.x.x)
+ - yarn  (^v0.22.0)
+ - python (^v3.4)
+ - composer  (^v1.4.2)
+ - mysql (or other database)
+
+####  Installation
+
+##### 1. Webapplicatie
+To install the webapplication, first go the the folder "webapp". Then use yarn to install the required modules.
+
+```
+$  cd webapp/
+$  yarn install
+```
+
+
+Copy .env.example to .env, and change the URL to the one of your API. This will probably be a localhost:// + the port where you run the API.
+
+```
+$  cp .env.example .env
+$  nano .env
+$  yarn start
+$  yarn watch css
```

Comment:
It's yarn watch-css.
But I'll change this later when you do yarn start, it automatically starts the watch-css